packages/react/src/components/ResourceList/ResourceList.jsx

```diff
@@ -141,6 +144,11 @@ class ResourceList extends Component {
           } ${prefix}--structured-list-row`}
           tabIndex={customAction ? -1 : idx}
           onClick={onRowClick ? () => onRowClick(id) : null}
+          onKeyDown={
+            onRowClick
+              ? handleSpecificKeyDown([keyboardKeys.ENTER], () => onRowClick(id))
+              : undefined
+          }
```

Comment:
Fixes a linting error that calls for onClick events to also have onKey* events with them.